### PR TITLE
[Tests-Only] Refactor api scenario outline examples (part3)

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

On top of https://github.com/owncloud/core/pull/38444
## Related Issue

- Part of https://github.com/owncloud/core/issues/38371

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by Talank — COMMENTED** [review]
LGTM

**Review by swoichha — COMMENTED** [review]
Please make sure that the expected to fail files are updated in ocis and cs3org/reva. Other than that evrything LGTM :+1:

File: tests/acceptance/features/apiWebdavMove2/moveFile.feature [hunk 1]
```diff
@@ -272,14 +272,17 @@ Feature: move (rename) file
...
+    When user "Alice" moves the following file using the WebDAV API
+      | source         | destination   |
+      | /textfile0.txt | *a@b#c$e%f&g* |
+      | /textfile1.txt | 1 2 3##.##    |
+    Then the HTTP status code of responses on all endpoints should be "201"
+    And the content of the following files for user "Alice" should be the following plus end-of-line
+      | filename       | content                   |
+      | *a@b#c$e%f&g*  | ownCloud test text file 0 |
+      | 1 2 3##.##     | ownCloud test text file 1 |
+    And the content of file "*a@b#c$e%f&g*" for user "Alice" should be "ownCloud test text file 0" plus end-of-line
```
Comment (individual-it) [inline tests/acceptance/features/apiWebdavMove2/moveFile.feature:285]: why checking that again its also checked in line 283

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/apiWebdavMove2/moveFile.feature (hunk 1 above)
Comment (HariBhandari07) [reply tests/acceptance/features/apiWebdavMove2/moveFile.feature]: forgot to remove the line.

**Review by individual-it — APPROVED** [review]
please remember to bump the core commit id in ocis and cs3org/reva

**Pull request merged**